core/src/main/java/me/mykindos/betterpvp/core/database/Database.java

```diff
@@ -51,25 +51,35 @@ public void executeUpdateAsync(Statement statement) {
         UtilServer.runTaskAsync(core, () -> executeUpdate(statement));
     }
 
+    public void executeUpdate(Statement statement) {
+        executeUpdate(statement, null);
+    }
+
     /**
      * @param statement The statement and values
      */
-    public void executeUpdate(Statement statement) {
+    public void executeUpdate(Statement statement, Consumer<ResultSet> callback) {
         Connection connection = getConnection().getDatabaseConnection();
         try {
             @Cleanup
-            PreparedStatement preparedStatement = connection.prepareStatement(statement.getQuery());
+            PreparedStatement preparedStatement = connection.prepareStatement(statement.getQuery(), java.sql.Statement.RETURN_GENERATED_KEYS);
```

Comment:
Is this the best way to do this? Will this cause problems elsewhere?

Reply:
It's viable, but not the best way. I typically just generate a UUID in code instead of using an auto-incremented row ID (there are cons to auto-incrementing)

See the ClanRepository for more details

Reply:
> It's viable, but not the best way. I typically just generate a UUID in code instead of using an auto-incremented row ID (there are cons to auto-incrementing)
> 
> See the ClanRepository for more details

This was in order to use the type used by the logs table, but I am thinking of converting that to use UUID. I also don't like how I log to the logs table, as I use my own sql statement. I think a better solution would be to modify Logger#log and just use that.